toucan_connectors/revinate/revinate_connector.py

```diff
@@ -0,0 +1,24 @@
+"""
+Revinate connector
+
+Documentation can be found at: https://porter.revinate.com/documentation
+
+This connector inherits from the HTTP connector classes rather than the base toucan-connector classes
+"""
+import pandas as pd
+
+from toucan_connectors.http_api.http_api_connector import HttpAPIConnector, HttpAPIDataSource
+
+
+class RevinateDataSource(HttpAPIDataSource):
+    base_url = 'https://porter.revinate.com'
+
+
+class RevinateConnector(HttpAPIConnector):
+    data_source_model: RevinateDataSource
+
+    username: str
+    password: str
```

Comment:
According to the docs, we will need an `api_key` instead of a `password` (https://porter.revinate.com/documentation#intro-auth)

Reply:
Yes, these are properties that come with the template... I haven't touched the template yet other than to add the base_url... I should have probably just erased these :+1: